spec/controllers/families_controller_spec.rb

```diff
@@ -25,4 +25,41 @@
       end
     end
   end
+
+  describe "PUT /families/:id" do
+    context "When the family id exist" do
```

Comment:
A suggestion to change the description: "when the family exists"

Reply:
I removed the unnecessary test as recommended. Thanks!